source/PokerSetup.py

```diff
@@ -0,0 +1,344 @@
+import sys
+
+PlayersStillInGameList = PlayerList
+BlindsList = PlayersStillInGameList
+PlayersPlayingRoundList = PlayersStillInGameList
+PlayersOutOfGameList = []
+
+class PokerGameSetup:
+
+    def __init__(self, name, money: int = 0):
+        self.name = []
+        self.hand = []
+        self.money = money
+
+    def preGame(self, players: PlayerList):
+        #   Give each player their money.
+        #
+        #   Option to have each player start with the same amount of money,
+        #   or with various amounts of money.
+        try:
+            Same = input('''Enter "Same" if each player starts with the same amount.
+Enter "Different" if each player start with different amounts: ''')
+
+            #   Each player starts with the same amount.
+            if Same == "Same" or Same == "same" or Same == "SAME" or Same == "S" or Same == "s":
+                Amount = int(input('No cents! Enter amount: $'))
+                for player in PlayerList:
+                    Player.addMoney(self, amount)
+
+            #   Each player starts with different amounts.
+            elif Same == "Different" or Same == "different" or Same == "DIFFERENT" or Same == "D" or Same == "d":
+                for player in PlayerList:
+                    Amount = int(input('No cents! Enter amount: $'))
+                    Player.addMoney(self, amount)
+
+            #   If a word other than "Same" or "Different" is entered.
+            else:
+                print('''YOU WERE SUPPOSE TO ENTER "Same" OR "Different"!!!
+YOU WANT TO BE DIFFICULT?!?!
+GO HOME!!!''')
+                sys.exit()
+
+            #   "Big Blind" amount.
+            Big_Blind = int(input('''Enter Big Blind amount.
+Must be an even number greater than 0: $'''))
+
+            #   If "Big Blind" amount is not even, or
+            #   is $0 then the question gets repeated.
+            while Big_Blind%2 != "1" or Big_Blind == "0":
+                Big_Blind = int(input('''Enter Big Blind amount.
+Must be an EVEN number greater than 0: $'''))
+
+        #   If an invalid value is entered (i.e. not a number).
+        except ValueError:
+            print(error)
+            sys.exit()
+
+        return Big_Blind
+
+    def Blinds(self):
+        #   Determines who is Big_Blind and who is Small_Blind.
+        #   Made a BlindsList to keep track.
+        #   BlindList will need to delete all players no longer playing.
+        #   Made a PlayersOutOfGameList, just keep the BlindList updated every time by deleting from PlayersOutOfGameList.
+        pass
+
+class PokerGameRounds:
+
+    def __init__(self, name, money: int = 0):
+        self.name = name
+        self.money = money
+        river = []
+
+    def startRound(self):
+
+        #   Shuffle deck!
+        Deck.shuffle(self)
+
+        #   Cuts
+        print('Cutting')
+        Deck.shuffle(self)
+
+        #   Deal cards!
+        Dealer.dealCards(2, PlayerList, deck)
+
+    def initialRiver(self):
```

Comment:
This can just call Jacobs river class into play?

Reply:
Yeah, as you said, most of it is just bare-bones stuff. I believe I did do valuable work creating the overall structure and approach. For the most part, I think people can just incorporate their functions straight into the structure. Obviously, some tweaking will have to be made, but for the most part, it's all there.